content/documentation.rst

```diff
@@ -62,20 +62,71 @@ semicolons, rather than after the Nth column.
     long lines.
 
 
-Writing User's Guides
-=====================
+About the Documentation Sections
+================================
+
+**Introduction**
+
+High level introduction to the software.
```

Comment:
this sentence does not communicate anything to me, it can be left off.

Instead I was expecting to see a few sencenses explaining that the documentatio is break up into section... why we break it up and what are those sections.

For each section we can have a few words describing them

Ex

Since SFTPPlus is a complex product and people using the product can come from various teams (sysadmin / support / operations)

* Introduction - general concept about the product
* Installation/Upgrade - informations for sysadmins
* Configuration - low level, reference documention for all configuration options
* Operations - high level concepts about using configuration to achieve something
* User Guides - detailed / step by step guides containing examples of performing various tasks
* Other pages - terminology, release notes. legal info... etc